pkg/virtctl/guestfs/guestfs.go

```diff
@@ -614,22 +605,26 @@ func createAttacher(client *K8sClient, p *corev1.Pod, command string) error {
 		"If you don't see a command prompt, try pressing enter.", resChan)
 }
 
-func (client *K8sClient) createInteractivePodWithPVC(pvc, image, ns, command string, args []string, isblock bool) error {
-	pod, err := createLibguestfsPod(pvc, image, command, args, kvm, isblock)
+func (client *K8sClient) createInteractivePodWithPVC(c *guestfsCommand, ns, command string, args []string, isblock bool) error {
```

Comment:
I see why you started passing `c *guestfsCommand` now. IMO placing these methods on the client is a bit awkward. What do you think about change the receiver to `c *guestfsCommand` and adding a func argument for the client instead?